dashboard/src/pages/packages/index.vue

```diff
@@ -43,8 +47,9 @@ const toggleLegend = () => {
     </h1>
 
     <div class="text-muted mb-3">
-      Showing {{ packageStore.packages.length }} /
-      {{ packageStore.packages.length }}
+      Showing {{ packageStore.page.offset + 1 }} -
+      {{ packageStore.lastResultOnPage }} of
+      {{ packageStore.page.total }}
     </div>
```

Comment:
It would be better to have it closer to the pagination from a development perspective too, to have everything together in a reusable Vue component.

Reply:
I did try to copy this text to right above the pager, but I couldn't get it to centre justify like the pager so it looked terrible.

I can try to move or copy close to the pager, but it will probably take some work to style it properly.

Reply:
I've repeated the "Showing packages X - Y of Z" underneath the pager.  After testing we can determine if this layout is good or if we want to adjust it further.